sdk/notificationhubs/notification-hubs/review/notification-hubs.api.md

```diff
@@ -28,26 +28,26 @@ export interface AdmNativeMessage {
 // @public
 export interface AdmNativeNotification {
     body?: string;
-    bodyLocArgs?: string[];
-    bodyLocKey?: string;
-    channelId?: string;
-    clickAction?: string;
+    body_loc_args?: string[];
+    body_loc_key?: string;
+    channel_id?: string;
+    click_action?: string;
     color?: string;
-    defaultSound?: boolean;
-    eventTime?: string;
+    default_sound?: boolean;
+    event_time?: string;
     icon?: string;
     image?: string;
-    localOnly?: boolean;
-    notificationCount?: number;
-    notificationPriority?: number;
+    local_only?: boolean;
+    notification_count?: number;
+    notification_priority?: "PRIORITY_UNSPECIFIED" | "PRIORITY_MIN" | "PRIORITY_LOW" | "PRIORITY_DEFAULT" | "PRIORITY_HIGH" | "PRIORITY_MAX";
     sound?: string;
     sticky?: boolean;
     tag?: string;
     ticker?: string;
     title?: string;
-    titleLocArgs?: string[];
-    titleLocKey?: string;
-    visibility?: number;
+    title_loc_args?: string[];
+    title_loc_key?: string;
+    visibility?: "VISIBILITY_UNSPECIFIED" | "PRIVATE" | "PUBLIC" | "SECRET";
 }
 
 // @public
```

Comment:
I was thinking since `JsonNotification` isn't an input type anywhere, maybe we should just directly splat `contentType: "application/json;charset=utf-8"` onto all the types that extend it and have them extend `NotificationCommon` directly? Less purity, but fewer types